reactDashboard/src/components/RegionDropdown/RegionDropdown.css

```diff
@@ -1,4 +1,23 @@
 .region-menu {
-    height: 250px;
-    overflow-y: scroll;
+  height: 250px;
+  overflow-y: scroll;
+  width: 220px;
+}
+
+.selected-region {
+  margin: 10px 0;
+  width: 220px;
+}
+
+.darkmode .selected-region {
+  color: #121212;
+  background-color: #c1def1;
+}
+
+.region-selector-row {
+  padding: 0 20px 0 10px;
+}
+
+.darkmode .region-selector-row {
+  background-color: #133a53;
```

Comment:
Where is the declaration of background-color that this block is overriding

Reply:
added a declaration for background-color to .region-selector-row